oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/manual/relation/service/ServiceRelationClientSideMetrics.java

```diff
@@ -60,8 +62,9 @@ public class ServiceRelationClientSideMetrics extends Metrics {
     private String destServiceId;
     @Setter
     @Getter
-    @Column(columnName = COMPONENT_ID, storageOnly = true)
-    private int componentId;
+    @Column(columnName = COMPONENT_IDS, storageOnly = true)
+    @ElasticSearch.Keyword
+    private IntList componentIds = new IntList(3);
```

Comment:
@kezhenxu94 @wankai123 This is what I referred to, so I added this new annotation to override type. IntList is mapping to text type field.

> The field can be [Keyword](https://www.elastic.co/guide/en/elasticsearch/reference/7.17/keyword.html), [Numeric](https://www.elastic.co/guide/en/elasticsearch/reference/7.17/number.html), [ip](https://www.elastic.co/guide/en/elasticsearch/reference/7.17/ip.html), [boolean](https://www.elastic.co/guide/en/elasticsearch/reference/7.17/boolean.html), or [binary](https://www.elastic.co/guide/en/elasticsearch/reference/7.17/binary.html).

ref https://www.elastic.co/guide/en/elasticsearch/reference/7.17/search-aggregations-bucket-terms-aggregation.html